Gemfile.lock

```diff
@@ -477,4 +480,4 @@ DEPENDENCIES
   xml-simple
 
 BUNDLED WITH
-   2.2.22
+   2.2.31
```

Comment:
please don't change this. if you like us to update bundler version, please [update rubygems_version instead](https://github.com/rubygems/rubygems.org/commit/b0a413fa55f575aa6df2900175b726caf8a71e9b) in a separate PR

Reply:
or made a PR https://github.com/rubygems/rubygems.org/pull/2869 and reverted the change d717012e